game/platform.cc

```diff
@@ -1,6 +1,30 @@
 #include "platform.hh"
 #include "fs.hh"
 
+#include <cstdlib>
+#include <exception>
+#include <string>
+
+#if (BOOST_OS_MACOS)
+#include <CoreFoundation/CoreFoundation.h>
+
+char * CFStringCopyUTF8String(CFStringRef aString) {
+	if (aString == nullptr) {
+		return nullptr;
+	}
+
+	CFIndex length = CFStringGetLength(aString);
+	CFIndex maxSize =
+	CFStringGetMaximumSizeForEncoding(length, kCFStringEncodingUTF8) + 1;
+	char *buffer = (char *)malloc((size_t)maxSize);
+	if (CFStringGetCString(aString, buffer, maxSize, kCFStringEncodingUTF8)) {
```

Comment:
I copy-pasted that function from somewhere after a few attempts at doing it manually with variable degrees of success.

It's a good point, but considering this is just a short string, I'm not sure how important it would really be.